lbry/extras/daemon/daemon.py

```diff
@@ -2382,7 +2427,7 @@ async def jsonrpc_claim_search(self, **kwargs):
                          [--new_sdk_server=<new_sdk_server>]
 
         Options:
-            --name=<name>                   : (str) claim name (normalized)
+            --name=<name>                   : (str) claim name (normalized_name)
```

Comment:
is `normalized_name` more meaningful documentation for the user than `normalized` in this case?

Reply:
nice catch lol, it is back to normalized